spec/file_io_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rspec'
+require 'CSV'
+require './lib/file_io'
+require './lib/item'
+require './lib/merchant'
+
+describe FileIo do 
+  it 'exists' do 
+    file_io = FileIo.new 
+
+    expect(file_io).is_a? FileIo
+  end
+
+  it 'returns an array' do 
+    mock_row = {
+      id: 12_345,
+      name: 'Widget',
+      description: 'Item desc',
+      unit_price: '12.23',
+      merchant_id: '12345'
+    }
+
+    allow(CSV).to receive(:foreach).and_yield(mock_row)
```

Comment:
I think the spacing here could use a refresh. Let's get together as a group on conventions and fix this for all test files.